utils/classes.py

```diff
@@ -0,0 +1,14 @@
+class User():
+    def __init__(self, username:str, name:str, password:str, email:str):
+        self.username = username
+        self.name = name
+        self.password = password
+        self.email = email
+    def self_dict( self ):
+         return {
+                   'username' : self.username,
+                   'name'  : self.name, 
+                   'password' : self.password,
+                   'email' : self.email
+                }
```

Comment:
why is this method useful?

Reply:
it's for convenience in the testing, especially if we have more properties on a user object, however none of us feel strongly about it - do you think we should get rid?